### DISCUSSION TITLE
Assign copyright based on usage rights

### DISCUSSION THREAD

**Discussion by adrielulanovsky** [guardian/grid#3474] [Done/closed]
When changing usage rights to Staff - Photographer, BBC would like that if the original image copyright is empty, then the platform automatically assigns copyright = StaffPhotographer.publication.

If The Guardian agrees with this, it could be implemented by changing the UsageRightsMetadataMapper in the shared codebase. If not, then a pluggable mechanism will have to be implemented (perhaps using the already existing UsageRightsConfigProvider?).

I'd like to have feedback from The Guardian to see if this is a desired functionality for both organizations.

**Comment by paperboyo** [comment]
Our Picture Desk confirmed this is fine (to fill empty Copyright with `publication`).

**Comment by adrielulanovsky** [comment]
PR Raised: https://github.com/guardian/grid/pull/3477
